src/core/Expander/ExpanderContent/ExpanderContent.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { Component, ReactNode } from 'react';
+import { default as styled } from 'styled-components';
+import classnames from 'classnames';
+import { withSuomifiDefaultProps } from '../../theme/utils';
+import { TokensProp, InternalTokensProp } from '../../theme';
+import { HtmlDiv, HtmlDivProps } from '../../../reset';
+import { baseStyles } from './ExpanderContent.baseStyles';
+import {
+  ExpanderConsumer,
+  ExpanderContentBaseProps,
+} from '../Expander/Expander';
+
+const baseClassName = 'fi-expander';
+const contentBaseClassName = `${baseClassName}_content`;
+const contentOpenClassName = `${contentBaseClassName}--open`;
+const noPaddingClassName = `${contentBaseClassName}--no-padding`;
+
+export interface ExpanderContentProps extends Omit<HtmlDivProps, 'id'> {
+  /** Content for expander */
+  children: ReactNode;
+  /** Remove padding from expandable content area (for background usage with padding in given container etc.) */
+  noPadding?: boolean;
```

Comment:
Maybe adding the `@default false` to see the value in the styleguidist.

Reply:
Added and also added `!!` for the condition to explicitly change undefined to false when not provided.